sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -230,16 +230,16 @@ documentSentiment.getSentences().forEach(sentenceSentiment ->
 String document = "Bonjour tout le monde";
 DetectedLanguage detectedLanguage = textAnalyticsClient.detectLanguage(document);
 System.out.printf("Detected language name: %s, ISO 6391 name: %s, score: %f.%n",
```

Comment:
what do you think about renaming `score` to `confidence score` ? not super important